src/features/unified-share-modal/flowTypes.js

```diff
@@ -155,7 +158,7 @@ export type sharedLinkType = {
 };
 
 export type collaboratorType = {
-    collabID: number,
+    collabID: number | string,
```

Comment:
This doesn't seem ideal -- is there a way to keep it as one type?

Reply:
I agree. Unfortunately, the API uses string IDs, and the internal USM uses number IDs, so we have to support both :(

Reply:
On second thought, I'll convert the IDs to numbers in `convertCollabsResponse()`. I can convert them back to strings in future API requests, if necessary.